### DISCUSSION TITLE
OIDC: Unsupported scope "address" requested (provider: Cloudflare)

### DISCUSSION THREAD

**Discussion by Timmmy** [photoprism/photoprism#4593] [Q&A] [answered]
I've configured Cloudflare (Access) as my identity provider.
It is however refusing authorization requests because an "address" scope is requested (which it does not support).
Address is however not listed in .well-known, nor in the Photoprism docs.

`scope=address+email+openid+profile`

Any ideas?

**Comment by Timmmy** [comment, marked as answer]
Fixed by setting the (undocumented) PHOTOPRISM_OIDC_SCOPES env variable

**Reply by lastzero** [reply]
Glad you found it! We haven't included it in our documentation yet so as not to confuse less experienced users (assuming the defaults will work for pretty much all providers). However, since there seem to be providers who don't support "address" (which I thought would never be a problem), we should document it.

**Reply by Timmmy** [reply]
Glad you had it in the code already!
FYI: AWS Cognito also does not seem to support it

**Reply by lastzero** [reply]
Our documentation has been updated accordingly:
- https://docs.photoprism.app/getting-started/advanced/openid-connect/#config-options
